wurst/math/Matrices.wurst

```diff
@@ -90,43 +89,42 @@ public function matrix2.row(int id) returns vec2
         default
             return ZERO2
 
-public function matrix2.rotate(angle angl) returns matrix2

```

Comment:
Removed this function which could be redundant or even useless since it represented common matrix multiplication.